cmd/site-controller/deploy-watch-all-ns.yaml

```diff
@@ -86,6 +93,14 @@ rules:
   - create
   - delete
   - update
+- apiGroups:
```

Comment:
I wonder if we should have separate yamls for site controller with and without policy?

Reply:
From a technical standpoint, this can be applied without issues, even if the CRD is not defined.
One thing that might be impacted though if we separate them, would be the operator.

Reply:
Ok, if it won't cause errors when those CRDs do not exist, then we don't need separate yaml.